libr/anal/fcn.c

```diff
@@ -1862,12 +1862,21 @@ R_API RAnalFunction *r_anal_fcn_find_name(RAnal *anal, const char *name) {
 	return NULL;
 }
 
+#define MAXBBSIZE (1024*1024*1)
```

Comment:
why? yet another MAXBBSIZE defined and yet another random limit... 

```
[775/1136] Compiling C object 'libr/anal/04634f8@@r_anal@sha/fcn.c.o'.
../libr/anal/fcn.c:1865: warning: "MAXBBSIZE" redefined
 #define MAXBBSIZE (1024*1024*1)
 
../libr/anal/fcn.c:20: note: this is the location of the previous definition
 #define MAXBBSIZE (16 * 1024)
```

Reply:
Also, I think there is already `anal.bb.maxsize` which should be used instead of hardcoding value.s